### ISSUE TITLE
Missings tabs modifier API

### ISSUE THREAD

**Issue by MangelMaxime** [Dzoukr/Feliz.Bulma#7]
Hello @Dzoukr,

I think the [tabs modifier API](https://bulma.io/documentation/components/tabs/) is missing.

```fs
    Bulma.tabs [
        tabs.IsCentered // tabs is unkown
    ]
```

**Comment by Dzoukr** [comment]
Hello @MangelMaxime,

thanks for comment, I'll look at it!

**Comment by Dzoukr** [comment]
Fixed in 1.7.0 - there was wrong name :)

**Comment by MangelMaxime** [comment]
Awesome thank you :)
